03.asynchronous/callback/no_error.js

```diff
@@ -0,0 +1,22 @@
+import sqlite3 from "sqlite3";
+const db = new sqlite3.Database(":memory:");
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  () => {
+    db.run(
+      "INSERT INTO books(title) VALUES(?)",
```

Comment:
`CREATE TABLE` とスタイルを揃えるなら以下のようにしたほうがよいと思います。

```suggestion
      "INSERT INTO books (title) VALUES (?)",
```

Reply:
修正しました。